web/themes/joinup/joinup_theme.theme

```diff
@@ -1,6 +1,42 @@
 <?php
-
 /**
  * @file
  * Functions to support theming in the Joinup theme.
  */
+
+use Drupal\image\Entity\ImageStyle;
+use \Drupal\file\Entity\File;
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function joinup_theme_preprocess_page(&$variables) {
+  // Set page title.
+  $request = \Drupal::request();
+  $route_match = \Drupal::routeMatch();
+  $title = \Drupal::service('title_resolver')->getTitle($request, $route_match->getRouteObject());
+  $variables['title'] = $title;
+}
+
+/**
+ * Implements hook_theme_preprocess_menu().
+ */
+function joinup_theme_preprocess_menu(&$variables) {
+  if ($variables['menu_name'] != 'account') {
+    return;
+  }
+  // Set the account logo.
+  $account = \Drupal::currentUser()->getAccount();
+  $user = user_load($account->id(), TRUE);
+  /** @var \Drupal\image\Plugin\Field\FieldType\ImageItem $photo */
+  $photo = $user->get('field_user_photo')->first();
+  $variables['profile_icon'] = '/themes/joinup/images/user-profile-icon.png';
```

Comment:
It's good to make sure that there is an icon appearing, but we should also make sure that the user cannot create an account without a picture or at least set a default picture if none is supplied.